taxi/views.py

```diff
@@ -20,28 +25,28 @@
     return render(request, "taxi/index.html", context=context)
 
 
-class ManufacturerListView(generic.ListView):
+class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     model = Manufacturer
     context_object_name = "manufacturer_list"
     template_name = "taxi/manufacturer_list.html"
     paginate_by = 5
 
 
-class CarListView(generic.ListView):
+class CarListView(LoginRequiredMixin, generic.ListView):
```

Comment:
The `CarListView` class correctly uses `LoginRequiredMixin` to ensure that only authenticated users can access the list view.